src/Button.story.tsx

```diff
@@ -0,0 +1,294 @@
+          <Button forceHoverState>Hover</Button>
+          <Button forceActiveState>Active</Button>
+          <Button forceFocusState>Focused</Button>
+          <Button disabled>Disabled</Button>
+        </VerticalButtonGroup>
+      </DemoSection>
+
+      <DemoSection
+        title="Secondary & Icon Buttons"
+        description="Secondary buttons can be used with primary buttons or as by themselves. Icon buttons are a good way to communicate more information about the button’s action."
+      >
+        <VerticalButtonGroup title="Secondary">
+          <Button>Rest</Button>
+          <Button forceHoverState>Hover</Button>
+          <Button forceActiveState>Active</Button>
+          <Button forceFocusState>Focused</Button>
+          <Button disabled>Disabled</Button>
+        </VerticalButtonGroup>
+
+        <VerticalButtonGroup
+          title="Buttons with icons"
+          buttonProps={{
+            icon: iconElement
+          }}
+        >
+          <Button>Rest</Button>
+          <Button forceHoverState>Hover</Button>
+          <Button forceActiveState>Active</Button>
+          <Button forceFocusState>Focused</Button>
+          <Button disabled>Disabled</Button>
+        </VerticalButtonGroup>
+
+        <VerticalButtonGroup
+          title="Icon only"
+          buttonProps={{
+            icon: iconElement
+          }}
+        >
+          <React.Fragment>
+            <Button css={{ marginRight: 6 }} icon={iconElement} />
+            <Button css={{ marginLeft: 6 }} fab icon={iconElement} />
+          </React.Fragment>
+          <React.Fragment>
+            <Button
+              css={{ marginRight: 6 }}
+              forceHoverState
+              icon={iconElement}
+            />
+            <Button
+              css={{ marginLeft: 6 }}
+              forceHoverState
+              fab
+              icon={iconElement}
+            />
+          </React.Fragment>
+          <React.Fragment>
+            <Button
+              css={{ marginRight: 6 }}
+              forceActiveState
+              icon={iconElement}
+            />
+            <Button
+              css={{ marginLeft: 6 }}
+              forceActiveState
+              fab
+              icon={iconElement}
+            />
+          </React.Fragment>
+          <React.Fragment>
+            <Button
+              css={{ marginRight: 6 }}
+              forceFocusState
+              icon={iconElement}
+            />
+            <Button
+              css={{ marginLeft: 6 }}
+              forceFocusState
+              fab
+              icon={iconElement}
+            />
+          </React.Fragment>
+          <React.Fragment>
+            <Button css={{ marginRight: 6 }} disabled icon={iconElement} />
+            <Button css={{ marginLeft: 6 }} disabled fab icon={iconElement} />
+          </React.Fragment>
+        </VerticalButtonGroup>
+      </DemoSection>
+
+      <DemoSection
+        title="Hidden Button"
```

Comment:
I feel like the term `hidden` is reserved for something that shouldn't be displayed _at all_.  Might I suggest we use a different term for this style of button?  I see the word `simple` being used in the description, and much prefer that.  I think it would make sense to use a property with a string literal union prop like `feel: 'normal' | 'simple'` rather than a boolean.
 

Perhaps this dovetails with another question, which is whether we'd consider "link" styled clickable things as being replaced by this kind of button?

cc @adamatorres

Reply:
Two things:

1. Love the `feel` or `variant` prop instead of a boolean `'hidden'`.
2. A clickable link; if we want it to have the same padding/spacing, then yes 🤷‍♂